src/components/AppBar/DonateButton/DonateButton.tsx

```diff
@@ -21,11 +22,12 @@ export const DonateButton = () => (
   </DonateButtonWrapper>
 );
 
-export const DonateButtonHeart = () => (
+export const DonateButtonHeart: React.FC<LinkProps> = ({ onClick }) => (
```

Comment:
tracking props are declared in `DonateButton.tsx` (and passed to both donate button variations), so i took `LinkProps` instead of `LinkButtonProps`